src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -1801,6 +1801,10 @@ ExpressionPtr Visitor::visit([[maybe_unused]] Parser::BuiltInCallContext* ctx) {
     return createUnary(&makeIsLiteralExpression);
   } else if (functionName == "isnumeric") {
     return createUnary(&makeIsNumericExpression);
+  } else if (functionName == "INT()") { 
+    return createUnary(&makeStrToIntExpression);
+  } else if (functionName == "DOUBLE()") { 
+    return createUnary(&makeStrToDoubleExpression);
```

Comment:
You shouldn't add this here ( where it would require to update the grammar and other nastinesses),
but to the `processIriFunctionCall` where the name of the functions should be
`http://www.w3.org/2001/XMLSchema#integer`
and `http://www.w3.org/2001/XMLSchema#double`. There exists a mechanims in this function to get rid of the common prefix, and you can also trigger the corresponding function for the names `...float`, `...int`, `...decimal`.
(we use those interchangeably in QLever at them moment, as we have only one floating point type and one integer type.

Reply:
Removed the part and added it to processIriFuntionCall(). Defined it similar to the part with the math functions, hope this is correct